src/test/java/roomescape/repository/ReservationsTest.java

```diff
@@ -0,0 +1,65 @@
+package roomescape.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import roomescape.domain.Reservation;
+import roomescape.domain.Reservations;
+
+class ReservationsTest {
+
+    private Reservations reservations;
+
+    @BeforeEach
+    void setUp() {
+        reservations = new Reservations(new ArrayList<>());
```

Comment:
```suggestion
    private final Reservations reservations = new Reservations(new ArrayList<>());;
```

junit5에서의 기본 라이프사이클은 테스트당이기 때문에 현재 케이스에서 beforeEach를 활용하지 않아도 됩니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle